build.sbt

```diff
@@ -6,21 +6,21 @@ description := "Scala client for Algolia Search API"
 
 version := "1.34.0"
 
-crossScalaVersions := Seq("2.11.11", "2.12.2")
+crossScalaVersions := Seq("2.11.12", "2.12.10", "2.13.1")
 
-scalaVersion := "2.12.2"
+scalaVersion := "2.13.1"
 
 coverageEnabled := false
 
-val asyncHttpClientVersion = "2.10.1"
-val json4sVersion = "3.5.2"
-val slf4jVersion = "1.7.25"
-val scalaUriVersion = "0.4.16"
+val asyncHttpClientVersion = "2.10.4"
+val json4sVersion = "3.6.7"
+val slf4jVersion = "1.7.30"
+val scalaUriVersion = "1.4.10"
```

Comment:
No BC upgrading to a new major version of this dependency?

Reply:
So far so good 🤞